qt_ui/windows/basemenu/QBaseMenu2.py

```diff
@@ -241,9 +241,47 @@ def closeEvent(self, close_event: QCloseEvent):
         GameUpdateSignal.get_instance().updateGame(self.game_model.game)
 
     def get_base_image(self):
+        try:
+            game = self.game_model.game
+        except:
+            game = None
+
         if self.cp.cptype == ControlPointType.AIRCRAFT_CARRIER_GROUP:
+            for ground_object in self.cp.ground_objects:
+                for group in ground_object.groups:
+                    for unit in group.units:
+                        if unit.type == "Forrestal":
+                            return "./resources/ui/carrier_cv59.png"
+                        elif (
+                            unit.type == "Stennis"
+                            and game is not None
```

Comment:
Why would this ever be `None`?

Reply:
I encountered at least one instance during testing when game was None during testing. I did not investigate it further, just added a check against it.

Reply:
That sounds like a bug. All this does is hide that bug.

Reply:
Do you want me to take the evaluation against None and the AttributeError checking above it away?

Reply:
Unless you can explain to me how this window is open without a game loaded, yes. That should be impossible. If it's happening that is a different bug that should be fixed.